portfolio/src/main/webapp/script.js

```diff
@@ -25,9 +25,22 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
 //add Hello Nzinga! to the page
-async function getHelloNzingaUsingAsyncAwait() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('HelloNzinga-container').innerText = quote;
+//async function getHelloNzingaUsingAsyncAwait() {
+//  const response = await fetch('/data');
+//  const quote = await response.text();
+//  document.getElementById('HelloNzinga-container').innerText = quote;
+//}
+
+function getHelloNzingaUsingAsyncAwait() {
+    fetch('/data')  // sends a request to /my-data-url
+    .then(response => response.json()) // parses the response as JSON
+    .then((myObject) => { // now we can reference the fields in myObject!
+    console.log(myObject[0]);
+    console.log(myObject[1]);
+    console.log(myObject[2]);
+    const quote = myObject[0]+" "+myObject[1]+" "+myObject[2];
+    document.getElementById('HelloNzinga-container').innerText = quote;
```

Comment:
This is not formatted correctly but it looks like it was replaced in the following PR so it's all good